1.9/administration/metrics/index.md

```diff
@@ -3,4 +3,142 @@ post_title: Metrics
+
+<!-- insert graphic -->
+
+Per-container metrics tags enable you to arbitrarily group metrics, for example on a per-framework or per-system or agent basis. Here are the available tags:
+
+* `agent_id`
+* `container_id`
+* `executor_id`
+* `framework_id`
+* `framework_name`
+* `framework_principal`
+* `hostname`
+* `labels`
+
+DC/OS applications will discover the endpoint via an environment variable (`STATSD_UDP_HOST` or `STATSD_UDP_PORT`). Applications leverage this StatsD interface to send custom profiling metrics to the system.
+
+## Security
+Because most metrics are sent to other service stacks and not consumed by DC/OS users, there is not any role based access control for them. However, the HTTP producer does expose and API endpoint, which can be consumed by DC/OS users. Because of this, Enterprise DC/OS provides coarse grained ACLs via the Admin Router proxy to ensure only DC/OS superusers have access to this HTTP API endpoint. 
+
+## Metrics reference
+These metrics are automatically collected.
+
+### Per-container resource resource utilization
+
+| Metric            | Description                  |
+|-------------------|------------------------------|
+| usage.Free        | Available capacity in bytes. |
+| usage.Total       | Available capacity in bytes. |
+| usage.Used        | Capacity used in bytes.      |
+| usage.InodesFree  | Available Inodes in btyes.   |
+| usage.InodesTotal | Available Inodes in bytes.   |
+| usage.InodesUsed  | Inodes used in bytes.        |
+
+###  Node
+
+#### Metrics
+   
+| Metric            | Description                  |
+|-------------------|------------------------------|
+| cpu.cores         |    Percentage of cores used.     |
+| cpu.idle         |     Percentage of CPUs idle.         |
+| cpu.system         |    Percentage of system used.   |
+| cpu.total         |   Percentage of CPUs used.  |
+| cpu.user         |   Percentage of CPU used by the user.   |
+| cpu.wait         |   Percentage idle while waiting for an operation to complete.    |
+| load.1min         |     Load average for the past minute.       |
+| load.5min         |   Load average for the past 15 minutes 5 minutes.        |
+| load.15min         |    Load average for the past 15 minutes.        |
+| memory.buffers         |   Number of memory buffers.     |
+| memory.cached         |   Amount of cached memory.   |
+| memory.free         |    Amount of free memory in bytes.   |
+| memory.total         |   Total memory in bytes.   |
+| processes         |  Number of processes that are running.          |
+| swap.free         |  Amount of free swap space.   |
+| swap.total         |  Total swap space.    |
+| swap.used         |    Amount of swap space used.    |
+| uptime          |   The system reliability and load average.    |
+   
+#### Filesystem
+   
```

Comment:
Perhaps add a mention that `{{.Name}}` is a part of a [go template](https://golang.org/pkg/html/template/) and will automatically be filled in based on the local filesystem's mount path (e.g., `/`, `/boot`, etc)?